devicon.json

```
@@ -7509,6 +7509,30 @@
         "color": "#2298BD",
         "aliases": []
     },
+    {
+        "name": "tauri",
+        "altnames": [],
+        "tags": [
```

I think we should add nodejs as a tag here

ref: https://tauri.app/about/intro
> The core is built with Rust, and the CLI leverages Node.js making Tauri a genuinely polyglot approach to creating and maintaining great apps.
```suggestion
        "tags": [
            "nodejs",
```